client/js/jsonedit.js

```diff
@@ -554,6 +554,21 @@ const jeCommands = [
       jeSetAndSelect(w.get('width'));
     }
   },
+  {
+    id: 'je_dragLimit',
+    name: 'drag limits',
+    context: '^[^ ]* ↦ dragLimit',
+    call: async function() {
+      const w = widgets.get(jeStateNow.id);
+      jeStateNow.dragLimit = {
+        minX: '###SELECT ME###',
+        maxX: 1600 - w.get('width'),
+        minY: 0,
+        maxY: 1000 - w.get('height'),
+      };
+      jeSetAndSelect(0);
+    }
+  },
```

Comment:
I don't know how to use these edit commands but this looks like it will only let you set one attribute of the drag limit which doesn't seem useful. I don't think you need this part of the change at all, but again not sure if you were intentionally trying to add to these commands.

Reply:
The idea is that if dragLimit is empty, you click the button and it fills it with all 4 default elements so you don't have to do the typing.  The defaults leave the widget visible on the screen.  But the problem is instead of the button disappearing, it stays and if you press it again, it overwrites what you have.  Any solutions or ideas @rogerl50 ?

Reply:
The updated PR looks like it resolves this. Thanks.